### PR TITLE
Design-teardown

### PR DESCRIPTION
**Pull request by Dipeshtwis**
This project includes :-

- used flexbox in the header section
- used grid to group all the elements on the Smashing homepage.
- used background colors of divs with different shades of gray which correspond to their visual weight on the Smashing homepage.
- use fonts from google font

### CODE REVIEW

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,575 @@
...
+
+      <div class="h1-content">
+        <h2 class="section-header">
+          <span>Don't Miss These Latest</span>
+          <br />
+          <span class="sec-line">Articles</span>
+        </h2>
+      </div>
+      <div class="header-grid">
+        <div class="header-author first-sec">
```
Comment (addod19) [inline index.html]: Kindly use ```<article> instead of <div class="header-author first-sec">```

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi **TEAM**, you did really great in building this project :+1: However, to improve our code and make it even better, let us address the comments below:
- ```Nav```, the nav items uses div to position the elements
ccp
- Kindly apply the DRY rule to our CSS file, we have some repeating properties, let us fix it. Let us use this [article](https://www.linkedin.com/in/daniel-larbi-addo/) as a guide
- Let us implement the float property in positioning some of our elements
wa
- Let us use article tag where appropriate
wa1

Kindly fix these changes and submit another code review :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by rida-elbahtouri** [comment]
**we added these changes to our project :**

the nav items use div to position the elements
apply the DRY rule to our CSS file
implement the float property  
use article tag

**Review by bolah2009 — APPROVED** [review]
It's a 👍🏾 from me!

### :heavy_check_mark: Approved!

Awesome job making the suggested changes. :relaxed: 

Good luck with your next projects. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

File: index.html (hunk 1 above)
Comment (Dipeshtwis) [reply index.html]: We used the article wherever requires.
Thank you for your suggestion.

**Pull request merged**